tests/test_bivariate_analysis.py

```diff
@@ -131,6 +134,7 @@ def test_code_export_verbosity_low():
     # Test code equivalence
     assert len(exported_code) == 1
     assert exported_code[0] == expected_code[0], "Exported code mismatch"
+    check_section_executes(bivariate_section, df=get_test_df())
```

Comment:
nitpick: there is a `Test code equivalence` which now seems to be related to this line

Reply:
Not sure what you mean.

Reply:
Oh I see what you mean. Added a blank line before to not make it look like the `check_section_executes` is part of the black which tests code equivalance.

Reply:
👍 😅